### DISCUSSION TITLE
Error parameter text = Labeling dalam pembuatan ggplot

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#844] [Q&A] [answered]
Dear Algoritma Team, izin bertanya . Saya terdapat error saat pembuatan interactive plot. berikut error nya: 
![image](https://github.com/teamalgoritma/community/assets/150925524/27397ef7-3285-4289-8590-135f1fa44e57)
Terimakasih

**Comment by kinantytasyao** [comment, marked as answer]
Dear @rastitiratnasari 

Berdasarkan error yang dilampikran caused by error adalah object 'labeling' pada parameter text di geom_point() not found. Karena parameter text di aes() mengakses data golds_1, kemungkinan besar kolom labeling belum disimpan pada data golds_1 atau Ibu typo memanggil kolom labeling ini. Silakan pastikan apakah sudah terdapat kolom `labeling` pada data golds_1 untuk menyelesaikan error ini dan tidak terdapat typo pada penulisan code.

Demikian yang dapat kami sampaikan, jika jawaban ini membantu silakan mark as answer :)
